.github/workflows/bundles-linux.yml

```diff
@@ -0,0 +1,113 @@
+        run: sudo apt-get install xdg-utils
+
+      - name: Install test packages
+        if: ${{ inputs.test }}
+        run: |
+          sudo apt-get install xvfb tigervnc-standalone-server tigervnc-common
+
+      - uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: ${{ inputs.java-distro }}
+          java-version: ${{ inputs.java-version }}
+
+      - name: Setup JavaFX
+        run: |
+          wget -P /tmp https://download2.gluonhq.com/openjfx/17/openjfx-17_linux-x64_bin-jmods.zip
+          unzip /tmp/openjfx-17_linux-x64_bin-jmods.zip -d /tmp
+
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: Run Tests
+        if: ${{ inputs.test }}
+        run: |
+          export DISPLAY=:90
+          mkdir /home/runner/.vnc
+          echo 123456 | vncpasswd -f > /home/runner/.vnc/passwd
+          chmod -v 600 /home/runner/.vnc/passwd
+          vncserver :90 -localhost -nolisten tcp
+          mvn -B -ntp clean verify checkstyle:checkstyle --no-transfer-progress
+          vncserver -kill :90
+
+      - name: Build and package JAR
+        run: |
+          mvn -B -ntp -q versions:set -DnewVersion=${{ env.VERSION }} -DgenerateBackupPoms=false
+          mvn -B -ntp -q clean package -Dmaven.test.skip=true -X
+        env:
+          VERSION: ${{ inputs.version }}
+
+      - name: Create Bundle using JPackage
+        run: |
+          # Create DEB
+          .github/scripts/jpackage.sh \
+            --icon app/assets/linux/icon-linux.png \
+            --java-options '"-Djdk.gtk.version=2"' \
+            --java-options '"--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED"' \
+            --java-options '"-Djava.library.path=/opt/scenebuilder/lib/runtime/bin:/opt/scenebuilder/lib/runtime/lib"' \
+            --linux-menu-group '"Development;Building;GUIDesigner;Java;"' \
+            --resource-dir app/assets/linux \
+            --install-dir /opt \
+            --type deb
+          mv ${{ env.INSTALL_DIR }}/*.deb ${{ env.INSTALL_DIR }}/SceneBuilder-${{ env.TAG }}.deb
+          # Create RPM
+          .github/scripts/jpackage.sh \
+            --icon app/assets/linux/icon-linux.png \
+            --java-options '"-Djdk.gtk.version=2"' \
+            --java-options '"--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED"' \
+            --linux-menu-group '"Development;Building;GUIDesigner;Java;"' \
+            --resource-dir app/assets/linux \
+            --install-dir /opt \
+            --type rpm
+          mv ${{ env.INSTALL_DIR }}/*.rpm ${{ env.INSTALL_DIR }}/SceneBuilder-${{ env.TAG }}.rpm
+          ls ${{ env.INSTALL_DIR }}
+        env:
+          MAIN_CLASS: com.oracle.javafx.scenebuilder.app.SceneBuilderApp
+          JAVAFX_HOME: /tmp/javafx-jmods-17/
```

Comment:
here and below, I see a number of references to 17. We can either use 17.0.2 or 18-ea. 
The next release of Scene Builder will be when JavaFX 18 is released, so maybe we should already depend on 18-ea?

Reply:
This value could also be made configurable form the caller workflow, just like `JAVA_VERSION` is.

Reply:
@johan recall it's not just the JAVAFX_HOME directory that needs update but also the URL used to download the mods.

Reply:
I would configure `javafx-version` as an input in `bundles-kit.yml` just like `java-version`. 

Download link can then it. Linux example:

> https://download2.gluonhq.com/openjfx/{javafx-version}/openjfx-{javafx-version}_linux-x64_bin-jmods.zip